library/src/main/java/pl/netigen/rewards/RewardParams.java

```diff
@@ -0,0 +1,40 @@
+package pl.netigen.rewards;
+
+import android.view.View;
+
+import androidx.annotation.DimenRes;
+import androidx.annotation.DrawableRes;
+import androidx.annotation.StringRes;
+
+import java.util.List;
+
+class RewardParams {
+
+    @StringRes
+    int buttonPositiveTextResId = 0;
+    @StringRes
+    int rewardDescrpitonTextResId = 0;
```

Comment:
Misspeling:
"rewardDescrpitonTextResId"  should be rewardDescriptionTextResId

Reply:
Done